demos/browser/app/meetingV2/styleV2.scss

```diff
@@ -801,12 +801,13 @@ a.markdown:active {
   .v-grid.featured.size-25,
   .v-grid.featured.size-26,
   .v-grid.featured.size-27 {
-    grid-template: repeat(8, 1fr) / repeat(4, 1fr);
+    grid-template: repeat(9, 1fr) / repeat(7, 1fr);
```

Comment:
Since this is for smaller screen, can we do `repeat(9, 1fr) / repeat(6, 1fr);` instead and feature occupied 4 rows as before.